riff-raff/app/conf/context.scala

```diff
@@ -219,6 +219,26 @@ class Config(configuration: TypesafeConfig, startTime: DateTime) extends Logging
     }
   }
 
+  object management {
+    object aws {
+      // TODO read config from more generic location
```

Comment:
Are these for later? Agree that these aren't perhaps the best place to grab them from long term.

Reply:
Yes, will re-arrange config in a follow-up PR. Should have made that obvious, apologies.

Reply:
https://github.com/guardian/riff-raff/pull/697